.jenkins/pytorch/test.sh

```diff
@@ -307,12 +308,27 @@ test_distributed() {
 
 test_rpc() {
   if [[ "$BUILD_ENVIRONMENT" != *rocm* ]]; then
-    echo "Testing RPC C++ tests"
+    if [[ -n "$IN_WHEEL_TEST" ]]; then
+      echo "Testing RPC C++ tests with the install folder"
+      # Rename the build folder when running test to ensure it
+      # is not depended on the folder
+      mv "$BUILD_DIR" "$BUILD_RENAMED_DIR"
```

Comment:
Where is $BUILD_RENAMED_DIR defined? I'm failing to find it in your changes--does it come from exisitng code?

Reply:
yes it was from the [1/n] PR

Reply:
Would these be better served as a `ghstack` PR so we can get a full lineage of these changes?

Just reference for the future

Reply:
> Would these be better served as a `ghstack` PR so we can get a full lineage of these changes?
> 
> Just reference for the future

Thanks, will do in next diffs